lms/djangoapps/ccx/views.py

```diff
@@ -237,14 +237,24 @@ def create_ccx(request, course, ccx=None):
     # Save display name explicitly
     override_field_for_ccx(ccx, course, 'display_name', name)
 
-    # Hide anything that can show up in the schedule
+    is_course_licensing_enable = configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False)
+    # if license enforcement is enabled and user is allowed to create ccx, then all units will be shown in schedule.
+    hide = not (
+        is_course_licensing_enable and run_extension_point(
+            'PCO_IS_USER_ALLOWED_TO_CREATE_CCX',
+            user=request.user,
+            master_course=course.id,
+        )
+    )
```

Comment:
I don't thing we are adding more params to this run_extension_point, what do you think about this?. @anfbermudezme 

```suggestion
    hide = not (
        is_course_licensing_enable and
        run_extension_point('PCO_IS_USER_ALLOWED_TO_CREATE_CCX', user=request.user, master_course=course.id)
    )
```